tests/python/test_offline_cache.py

```diff
@@ -327,12 +327,12 @@ def compute_y():
             enable_fallback=False,
             **current_thread_ext_options())
     assert added_files(curr_arch) == expected_num_cache_files(
-        curr_arch, [1] * 8)
+        curr_arch, [1] * 9)
```

Comment:
why did this change btw?

Reply:
Since a `self.loss.grad.fill(1.0)` is added, an additional kernel call is taken into count.